docs/CLI.md

```diff
@@ -180,8 +180,6 @@ Alias: `--collectCoverage`. Indicates that test coverage information should be c
 
 Indicates which provider should be used to instrument code for coverage. Allowed values are `babel` (default) or `v8`.
 
-Note that using `v8` is considered experimental. This uses V8's builtin code coverage rather than one based on Babel. It is not as well tested, and it has also improved in the last few releases of Node. Using the latest versions of node (v14 at the time of this writing) will yield better results.
```

Comment:
Please remove this paragraph in version 29. In all other versions (28, 27, 26, 25) turn it into a `:::note`.

Reply:
Done!